js/script.js

```diff
@@ -1,16 +1,14 @@
 const setUpAbout = () => {
-  $(".banner-about").click(() => {
-    if ($(".about-text-popup").css("display") === "none") {
-      $(".about-text-popup").css("display", "block");
-    } else {
-      $(".about-text-popup").css("display", "none");
-    }
+  const aboutElement = document.querySelector(".about-text-popup");
+  document.querySelector(".banner-about").addEventListener("click", () => {
+    aboutElement.style.display =
+      aboutElement.style.display !== "block" ? "block" : "none";
```

Comment:
The original logic had a bug. On startup, `display` is set to the empty string. We now catch that.